src/components/SingleSpliceChart/index.tsx

```diff
@@ -16,27 +17,27 @@ const singleSplineStyle = cva(
   {
     variants: {
       variant: {
-        single_spline_area_chart: "",
+        [singleSpliceType.singleSpliceChart]: "",
       },
       size: {
-        sm: "px-4 py-2 text-sm",
-        md: "px-4 py-2 text-base",
-        lg: "px-6 py-3 text-lg",
+        [singleSpliceSize.small]: "px-4 py-2 text-sm",
+        [singleSpliceSize.medium]: "px-4 py-2 text-base",
+        [singleSpliceSize.large]: "px-6 py-3 text-lg",
       },
       colorscheme: {
         primary: "text-black",
       },
     },
     compoundVariants: [
       {
-        variant: "single_spline_area_chart",
+        variant: singleSpliceType.singleSpliceChart,
         colorscheme: "primary",
         className: "text-[#DC2626] ",
       },
     ],
     defaultVariants: {
-      variant: "single_spline_area_chart",
-      size: "md",
+      variant:singleSpliceType.singleSpliceChart,
+      size: singleSpliceSize.medium,
       colorscheme: "primary",
```

Comment:
defined the name in enum for primary secondary etc etc

Reply:
ok